### PR TITLE
Alecb/generation api upgrade preannounce

### PR DESCRIPTION
**Pull request by alec-baird**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: documentation/breaking-changes/upcoming-breaking-changes.md [hunk 1]
```diff
@@ -27,3 +27,33 @@
 
 # Upcoming Breaking Changes
 
+    ## Release 13.0.0 - November 2024
+
+    The following cmdlets were affected this release:
+
+    **All Cmdlets**
+    - All cmdlets with SystemData properties have their property-type switched from struct to string.
```
Comment (msJinLei) [inline documentation/breaking-changes/upcoming-breaking-changes.md:35]: Please refer to https://github.com/Azure/azure-powershell/blob/3dfb95de6647a49c2ed584abe18195f3975dc33e/tools/StaticAnalysis/Exceptions/Az.DesktopVirtualization/BreakingChangeIssues.csv to preannounce all the non-v4-upgrading breaking changes

File: documentation/breaking-changes/upcoming-breaking-changes.md [hunk 2]
```diff
@@ -27,3 +27,33 @@
 
 # Upcoming Breaking Changes
```
Comment (msJinLei) [inline documentation/breaking-changes/upcoming-breaking-changes.md:28]:
```suggestion
# Upcoming Breaking Changes of Az.DesktopVirtualization
```

File: documentation/breaking-changes/upcoming-breaking-changes.md [hunk 3]
```diff
@@ -27,3 +27,33 @@
...
 
+    ## Release 13.0.0 - November 2024
```
Comment (msJinLei) [inline documentation/breaking-changes/upcoming-breaking-changes.md:30]:
```suggestion
    ## Release Az 13.0.0 - November 2024
```

File: documentation/breaking-changes/upcoming-breaking-changes.md [hunk 4]
```diff
@@ -27,3 +27,33 @@
...
+
+    The following cmdlets were affected this release:
```
Comment (msJinLei) [inline documentation/breaking-changes/upcoming-breaking-changes.md:32]:
```suggestion
    The cmdlets contain the following properties in their inputs and outputs are affected by the release :
```

File: documentation/breaking-changes/upcoming-breaking-changes.md [hunk 5]
```diff
@@ -27,3 +27,33 @@
...
+
+    **All Cmdlets**
```
Comment (msJinLei) [inline documentation/breaking-changes/upcoming-breaking-changes.md:34]:
```suggestion
```

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/DesktopVirtualization/DesktopVirtualization.Autorest/README.md [hunk 6]
```diff
@@ -103,4 +103,10 @@ directive:
       parameter-name: Force
     set:
       parameter-description: 'Specify to force userSession deletion.'
+  - where:
+      verb: (.*)
```
[the remark this answers is not in the document]
Comment (alec-baird) [reply src/DesktopVirtualization/DesktopVirtualization.Autorest/README.md:107]: You said you were doing the preannouncement for v4 upgrade, so I didn't include it here.

File: documentation/breaking-changes/upcoming-breaking-changes.md [hunk 7]
```diff
@@ -27,3 +27,33 @@
...
+    # New
+    # Sample of how the cmdlet should now be called
+    New-AzWvdWorkspace -Name <String> -ResourceGroupName <String> [-SubscriptionId <String>]
+     [-ApplicationGroupReference <String[]>] [-Description <String>] [-FriendlyName <String>]
+     [-IdentityType <String>] [-Kind <String>] [-Location <String>] [-ManagedBy <String>]
+     [-PlanName <String>] [-PlanProduct <String>] [-PlanPromotionCode <String>] [-PlanPublisher <String>]
+     [-PlanVersion <String>] [-PublicNetworkAccess <String>] [-SkuCapacity <Int32>]
+     [-SkuFamily <String>] [-SkuName <String>] [-SkuSize <String>] [-SkuTier <String>] [-Tag <Hashtable>]
+     [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
+    ```
```
[the remark this answers is not in the document]
Comment (alec-baird) [reply documentation/breaking-changes/upcoming-breaking-changes.md:59]: You said your team was handling the preannouncement for the v4 upgrade.

File: src/DesktopVirtualization/DesktopVirtualization.Autorest/README.md [hunk 8]
```diff
@@ -103,4 +103,46 @@ directive:
       parameter-name: Force
     set:
       parameter-description: 'Specify to force userSession deletion.'
+  - where:
+      verb: Register
+    set:
+      preview-announcement:
+        preview-message: This release will contain numerous breaking changes, please view the preannouncement here (https://github.com/Azure/azure-powershell/blob/main/documentation/breaking-changes/upcoming-breaking-changes.md)
```
Comment (msJinLei) [inline src/DesktopVirtualization/DesktopVirtualization.Autorest/README.md:110]:
```suggestion
        preview-message: This release will contain numerous breaking changes, please view the preannouncement here (https://go.microsoft.com/fwlink/?linkid=2292409)
```

**Review by msJinLei — CHANGES_REQUESTED** [review]
Please change the link in preannouncement to https://go.microsoft.com/fwlink/?linkid=2292409

File: documentation/breaking-changes/upcoming-breaking-changes.md [hunk 9]
```diff
@@ -25,5 +25,106 @@
     https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
 -->
 
-# Upcoming Breaking Changes
+# Upcoming Breaking Changes of Az.DesktopVirtualization
```
Comment (msJinLei) [inline documentation/breaking-changes/upcoming-breaking-changes.md:29]: Could you remove the file as the content is moved to the new PR https://github.com/Azure/azure-powershell/pull/26269

**Review by msJinLei — COMMENTED** [review]
Please remove the "upcoming-breaking-changes.md" and regenerate the module and commit all the necessary changes

**Review by msJinLei — APPROVED** [review]

**Pull request merged**